Azure Services/Azure Monitor/Agents/Migration Tools/DCR Config Generator/WorkspaceConfigToDCRMigrationTool.ps1

```diff
@@ -101,6 +101,13 @@ function Get-DCRFromWorkspace
         $linuxDCRTemplateParams | Out-File "$($FolderPath)/dcr_linux_arm_template_$currentDateTime.parameters.json"
         $linuxDCRArmTemplate | Out-File "$($FolderPath)/dcr_linux_arm_template_$currentDateTime.json"
     }
+
+    if (Find-IfVmiEnabled -ResourceGroupName $ResourceGroupName -WorkspaceName $WorkspaceName){
```

Comment:
Instead of providing both the DCRs, cant we just ask the user to give that input?